README.md
# Homsai Engine: maximum comfort, minimum consumption thanks to AI

**Homsai Engine** uses artificial intelligence to optimize the consumption of your home and save energy.
Thanks to our add-on you will no longer have to think about managing the devices in the house because it will be the Homsai Engine **that will make the photovoltaic system and the smart devices work in synergy**, guaranteeing savings and maximum comfort.
<br/><br/>

<p align="center">
  <img src="https://homsai.app/wp-content/uploads/2022/06/homsai1.gif" />
</p>
<br/><br/>

## Why choose Homsai Engine

- **Minimum effort, maximum result**: with Homsai Engine you will take advantage of a sophisticated algorithm that will allow you to spend nothing on energy, without the need to call an installer.
- Automatic detection of **all devices** on your instance
- Reporting with **detailed consumption** of smart devices
- **Simple** and well documented installation
  Add the add-on and proceed with the initial wizard: in a few minutes you can make your home really smart. With lower bills and maximum comfort for you and your family.

## Resources

- [GitHub](https://github.com/HOMSAI-Smart-Home/HOMSAI-Add-on)
- [Website](www.homsai.app)
- [Instagram](https://www.instagram.com/homsai.app/)
- [Facebook](https://www.facebook.com/HomsaiApp)
- [Contact Support](mailto:hello@homsai.app)


homsai/CHANGELOG.md
## 0.8.3

First public release.
Implements photovoltaic optimizations in HVAC devices:

- heating
- cooling

## 0.8.3.1

Now optimizer is enabled after init

## 0.8.3.2

Fixes bug on update HVAC device temperature while winter mode on

## 0.8.3.3

Add support for Heat Pump trigger (switch) during HVAC Init and PV Optimizer

## 0.8.3.5

Fixes a bug on Heat Pump trigger (switch) during PV Optimizer, reduced floor_radiant minimum exec time to 90 mins

## 0.8.6

Now Homsai requires username and password to login

## 0.8.7

Various bug fix


homsai/rootfs/var/www/web/main.dart.js_3.part.js
self.$__dart_deferred_initializers__=self.$__dart_deferred_initializers__||Object.create(null)
$__dart_deferred_initializers__.current=function(a,b,c,$){var A={
byi(){B.D8("it")
return new A.zN()},
zN:function zN(){}},C,B
A=a.updateHolder(c[4],A)
C=c[5]
B=c[0]
A.zN.prototype={
gR2(){return"Ok"},
gnG(d){return"Conferma"},
gRX(){return"Riprova"},
gjk(d){return"Avanti"},
ga3y(d){return"Indietro"},
git(d){return"Annulla"},
ga4T(){return"Testo copiato"},
S1(d){return d+"\xb0"},
gBk(d){return"Salva"},
ga9k(){return"Email non valida"},
ga9m(){return"Password non valida"},
ga9n(){return"Nome casa non \xe8 valido"},
ga9l(){return"Mail non riconosciuta. Registrati alla Closed Beta e attendi la mail di conferma."},
gacQ(){return"Grazie per esserti iscritto. Il tuo account \xe8 in attesa di approvazione. Nei prossimi giorni riceverai una mail con le credenziali dell'accesso"},
gacR(){return"Controlla la casella di posta elettronica per la trovare le credenziali d\u2019accesso"},
gaaW(d){return"Password"},
ga7a(){return"Password dimenticata?"},
gHI(d){return"Registrati"},
gaa0(){return"Accedi"},
gac1(){return"Reimposta password"},
gTU(){return"impostazioni"},
ga3a(){return"Hai gi\xe0 un account?"},
gabF(){return"Effettua l'accesso"},
gaa1(){return"bentornato\na %1casa%1"},
gabE(){return"rendi\nla tua casa\n%1davvero%1\nintelligente"},
gQY(){return"Non hai un account?"},
gac2(){return"reimposta\nla %1password%1"},
gSo(){return"Inserisci URL istanza"},
gacN(){return"Inserisci URL"},
gSn(){return"Gli URL locale e remoto servono a rendere raggiungibile il tuo impianto Home Assistant.\n%1Scopri di pi\xf9%1"},
ga7N(){return"Connettiti al WI-FI!"},
ga7M(){return"Per individuare l\u2019istanza locale di *Home Assistant* \xe8 necessario che lo smartphone sia connesso alla stessa Wi-Fi dell\u2019hub domotico."},
ga7S(){return"Ricerca istanza locale"},
ga7T(){return"Nessuna connessione rilevata"},
ga7U(){return"Ricerca..."},
ga7O(){return"Scansiona"},
ga7P(){return"Url non valido, assicurati che rispetti il formato http://XX.XXX.XXX.XXX:XXXX"},
ga7R(){return"L\u2019istanza non risulta raggiungibile. Verifica da Home Assistant e riprova"},
ga7Q(){return"L\u2019indirizzo fornito non corrisponde a nessuna istanza Home Assistant"},
ga2C(){return"Dai un nome alla casa"},
gNv(){return"Modifica casa"},
ga2x(){return y.c},
ga2w(){return y.c},
ga2B(){return"Nome casa"},
ga2A(){return"Coordinate GPS"},
ga2z(){return"Perch\xe9 abbiamo bisogno della posizione?"},
ga2y(){return"Homsai utilizza la posizione per ricavare i dati meteorologici per prevedere la variazione dei consumi e della produzione del fotovoltaico."},
ga9h(){return"benvenuto!"},
ga9i(){return"accedi e configura"},
ga9j(){return"comfort e risparmio"},
ga99(){return"Homsai \xe8 l'applicazione che ti consente di *ottimizzare* la gestione della tua smart home."},
ga9a(){return"Effettua la registrazione a Homsai con le tue credenziali di %l1Home Assistant%l1 e in pochi istanti avrai la tua smart home configurata."},
ga9b(){return"Grazie ad Homsai la tua casa si prender\xe0 cura di te *ottimizzando* i consumi e permettendoti un notevole *risparmio* in bolletta."},
ga9d(){return"Grazie alla nostra *intelligenza artificiale* riceverai suggerimenti personalizzati per *l\u2019efficienza e il risparmio*. Con un semplice click Homsai si occuper\xe0 di tutto."},
ga9e(){return"Utilizziamo %l2le pi\xf9 avanzate tecnologie%l2 per la salvaguardia dei dati. La tua privacy \xe8 al sicuro."},
ga9f(){return"L\u2019interfaccia intuitiva ti permetter\xe0 inoltre di controllare *tutti i tuoi dispositivi* e impostare *routine* facilmente."},
ga96(){return"Come funziona?"},
ga97(){return"Quali vantaggi ho?"},
ga98(){return"Voglio provare Homsai"},
ga9g(){return"Salta"},
ga9c(){return"Step %1 di 3"},
ga7G(){return"https://www.home-assistant.io"},
ga8v(){return"https://homsai.app"},
ga37(){return"Connessione internet assente"},
ga36(){return"Le funzionalit\xe0 non sono disponibili riconnettiti per continuare ad usare l\u2019app."},
ga35(){return"Connessione Home Assistant assente"},
ga34(){return"Le funzionalit\xe0 non sono disponibili riconnettiti dalle impostazioni per continuare ad usare l\u2019app."},
ga7X(){return"Consumo di ieri"},
ga8h(){return"Consumo ottimizzato"},
ga8i(){return"Energia Acquistata"},
ga8g(){return"Energia Immessa"},
ga7V(){return"--"},
ga7W(){return"Saldo"},
ga8f(){return"Guadagno con Homsai"},
ga8e(){return"Ottimizzazione consumi"},
ga83(){return"Cosa stai vedendo"},
ga82(){return"Il widget dei Consumi Ottimizzati mostra la differenza dei consumi all'interno di una giornata con e senza ottimizzazione, scoprendo il vantaggio economico di utilizzare Homsai."},
ga85(){return"Consumo attuale"},
ga84(){return"Mostra l'andamento dei consumi reali della tua casa nella giornata di ieri misurati con i sensori di consumo configurati."},
ga87(){return"Consumo ottimizzato"},
ga86(){return'Sulla base dei consumi mostrati in "Consumo attuale", Homsai mostra la stima dei valori ottenibili grazie alla sua ottimizzazione dei consumi, tenendo conto del meteo per efficentare la gestione della climatizzazione e prevedere la produzione di energia. \nGrazie ai suoi algoritmi, Homsai \xe8 in grado di migliorare nel tempo l\u2019efficenza di tale ottimizzazione, sfruttando sempre pi\xf9 l\u2019energia prodotta dai pannelli fotovoltaici e utilizzando meno energia acquistata.'},
ga89(){return"Legenda"},
ga88(){return"Linee"},
ga7Y(){return": Energia prodotta dai pannelli solari"},
ga7Z(){return": Energia totale consumata"},
ga8a(){return"Aree"},
ga8_(){return": energia prodotta residua"},
ga80(){return": consumo di energia prodotta"},
ga81(){return": consumo di energia acquistata"},
ga8b(){return"Saldo = Energia Immessa* - Energia Acquistata*"},
ga8c(){return"Guadagno con Homsai = Saldo stimato - Saldo attuale"},
ga8d(){return"*Il costo dell\u2019energia \xe8 calcolato sul prezzo medio dell\u2019energia considerando 0.35\u20ac per quella acquistata e 0.04\u20ac per l\u2019energia immessa."},
gFy(){return"kW"},
gOP(){return"Pannelli solari"},
gOM(){return"Consumo"},
ga5N(){return"Disponibile"},
gON(){return"Autoconsumo"},
gOO(){return"Acquistata"},
ga5O(){return"Recupero dati..."},
ga5M(){return"Non riusciamo a recuperare i dati del grafico, controlla nelle impostazioni di aver inserito i dati corretti sui sensori."},
ga2K(){return"Configura impianto"},
gRD(){return"Sensore Produzione"},
gabf(){return"Scegli Sensore di produzione"},
gOe(){return"Sensore Consumo"},
ga4O(){return"Scegli Sensore di consumo"},
gRx(){return"Data installazione fotovoltaico"},
gHw(){return"MM/AAAA"},
gHx(){return"Potenza nominale fotovoltaico"},
gRy(){return"kWp"},
gaaY(){return"Data non valida"},
gaaZ(){return"Completa correttamente la data"},
gade(){return"A"},
gvH(){return"Nessun sensore selezionato"},
gaau(){return"M"},
ga3B(){return"Sensore Batteria"},
ga3A(){return"Scegli batteria fotovoltaico"},
ga8x(){return"Sensore HVAC"},
ga8w(){return"Seleziona un sensore"},
gVx(){return"Consenso caldaia"},
gVw(){return"Seleziona un interuttore"},
gVv(){return"Seleziona il comando della caldaia per consentire ad Homsai di ottimizzare il riscaldamento."},
ga2J(){return"Configurazione incompleta"},
ga2H(){return"Assicurati di aver compilato correttamente tutti i campi.\nPer visualizzare correttamente i consumi ottimizzati completa la configurazione dell\u2019impianto.\nPotrai comunque completare la configurazione in un secondo momento dalla pagina Impostazioni."},
ga2I(){return"Campi incompleti:"},
gEN(){return"Ignora"},
gEO(){return"Modifica"},
gNw(){return"Modifiche non salvate. Vuoi ignorare e tornare indietro o continuare a modificare?"},
gaa3(){return"Esci"},
gaa2(){return"Vuoi davvero uscire da Homsai?\nUscendo, tutti i dati dell\u2019impianto verranno rimossi."},
gaa4(){return"Non e' possibile eseguire il logout in modo sicuro senza una connessione al tuo Home Assistant.\nDisconnettersi comunque?"},
ga43(){return"Optimizer disabilitato. Modificando la temperatura target lo abiliterai. vuoi procedere?"},
ga9X(){return"Inserisci l'Url locale"},
gabH(){return"Inserisci l'Url remoto"},
gIG(){return"Generale"},
ga29(){return"Segnala un problema"},
ga28(){return"Aiutaci a migliorare Homsai"},
gRi(){return"Impossibile raggiungere la pagina %1"},
ga5Z(){return"Elimina account"},
ga5Y(){return"Rimuovi il tuo account da Homsai"},
gTA(){return"Ottimizzazione autoconsumo"},
gTz(){return"Sfrutta al massimo l\u2019energia prodotta"},
gNn(){return"Email"},
ga2d(){return"Sensore di produzione"},
ga2a(){return"Sensore di consumo"},
ga27(){return"Sensore della batteria"},
ga2b(){return"Nome Impianto"},
ga2c(){return"Posizione Impianto"},
ga2e(){return"Versione App"},
gO6(){return"Clima"},
ga44(){return"- 0,5"},
ga45(){return"+ 0,5"},
gaaJ(){return"Altre impostazioni"},
ga62(){return"Inizializzazione..."},
ga46(){return"clima"},
ga4o(){return"Ottimizza dispositivo"},
ga47(){return"Consente di ottimizzare il dispositivo"},
ga4p(){return"Controllo manuale"},
ga48(){return"Passa in manuale fino alle 23:59"},
ga4q(){return"Limita orario"},
ga49(){return"Ottimizza clima solo in una certa fascia oraria"},
ga4r(){return"Dalle"},
ga4a(){return"8:00"},
ga4s(){return"Alle"},
ga4b(){return"8:00"},
ga4u(){return"Orario non valido"},
ga4t(){return"Per limitare il funzionamento dell\u2019ottimizzatore \xe8 necessario impostare sia l\u2019orario di inizio che l\u2019orario di fine. Controlla di aver impostato correttamente entrambi gli orari."},
ga4i(){return"La limitazione d\u2019orario non pu\xf2 coinvolgere pi\xf9 giorni.L\u2019orario di termine limitazione deve essere successivo all\u2019orario di inizio."},
ga4j(){return"Esempio valido:\nDalle: 8:00\nAlle: 20:00"},
ga4k(){return"Esempio non valido:\nDalle: 20:00\nAlle: 8:00"},
ga4m(){return"Imposta l'orario"},
ga4l(){return"Limite orario dalle"},
ga4n(){return"Limite orario alle"},
ga4h(){return"Ottimizzatore inattivo"},
ga4g(){return"Le modifiche alle impostazioni non avranno effetto fino alla riattivazione dell\u2019ottimizzatore."},
ga4f(){return"Ottimizza"},
ga4c(){return"Stai controllando "},
ga4d(){return"tutti i dispositivi clima."},
ga4e(){return"Orario non valido"},
ga8W(){return"Installa Add-On"},
ga8U(){return"L'add-on per Home Assistant permette di ottimizzare i consumi dovuti alla climatizzazione, in base alla produzione energetica dei pannelli fotovoltaici."},
ga8V(){return"Secgli tra le due opzioni di installazione e segui le istruzioni."},
ga8R(){return"Installazione automatica"},
ga8S(){return"Installazione manuale"},
ga8T(){return"Installa pi\xf9 tardi"},
gaa7(){return"Apri Home Assistant dal computer o dall'app per smartphone."},
gaa8(){return"Dal men\xf9 laterale, seleziona la voce 'Impostazioni'"},
gaa9(){return"Seleziona 'Componenti aggiuntivi', poi 'Raccolta di componenti aggiuntivi', in basso."},
gaaa(){return"Apri il men\xf9 in alto a destra, per poi selezionare 'Repository'."},
gaab(){return"Nella schermata che si aprir\xe0 incolla il seguente link e premi 'Aggiungi'."},
gaac(){return"\u201cHomsai Engine\u201d sar\xe0 aggiunto alla lista dei componenti locali."},
gaad(){return"Seleziona \u201cHomsai Engine\u201d e premi su \u201cInstalla\u201d dopodich\xe9 premi \u201cAvvia\u201d per attivare l\u2019add-on."},
ga3t(){return"Installazione in corso..."},
ga3u(){return"L'installazione dell'Homsai Engine pu\xf2 durare fino a 2 minuti. Mantieni l'app aperta per non perdere i progressi."},
ga3v(){return"Installazione completata"},
ga3s(){return"Installazione fallita"},
ga3p(){return"Verifica repository"},
ga3q(){return"Installazione add-on"},
ga3o(){return"Avvio add-on"},
ga3r(){return"Test"},
gGg(){return"Homsai Engine"},
ga33(){return"Configurazione sensori incompleta."},
ga32(){return"Configura"},
ga2X(){return"Nessun add-on trovato. Installa l\u2019add-on per poter utilizzare Homsai."},
ga2W(){return"Installa"},
ga3_(){return"\xc8 necessario inizializzare i dispositivi per poter avviare l\u2019ottimizzatore dei consumi."},
ga2Z(){return"Avvia"},
ga31(){return"Inizializzazione"},
ga30(){return"Inizializzazione dei dispositivi in corso..."},
ga2Y(){return"*Homsai Engine* non \xe8 raggiungibile."},
ga2V(){return"Dettagli"},
ga2U(){return"Avvia Homsai Engine"},
ga2T(){return"*Homsai Engine* \xe8 stato disattivato."},
ga2R(){return"Attiva"},
ga2S(){return"In attivazione..."},
ga38(){return"*Homsai Engine Optimizer* \xe8 stato disabilitato."},
ga8u(){return"homsai engine"},
ga8r(){return"Gestisci Add-on"},
ga8s(){return"Avvia installazione wizard"},
ga8t(){return"Add-on installato correttamente"},
ga8q(){return"inizializzazione"},
ga8p(){return"L\u2019inizializzazione \xe8 necessaria per identificare quali dispositivi contribuiscono attivamente alla climatizzazione.\nL\u2019inizializzazione impiega 20 minuti per ogni dispositivo, durante i quali non sar\xe0 possibile comandare il dispositivo manualmente."},
ga8k(){return"Inizializza dispositivi"},
ga8m(){return"Avvio\u2026"},
ga8l(){return"Inizializzazione in corso"},
ga8j(){return"Tempo stimato"},
ga8o(){return"Tempo rimanente"},
ga8n(){return"Dispositivi inizializzati"},
gTy(){return"autoconsumo"},
gTp(){return"CLIMA"},
gTu(){return"Ottimizza raffrescamento"},
gTv(){return"Sfrutta l'energia prodotta per raffrescare"},
gTq(){return"Configura raffrescamento"},
gTr(){return"Modifica le impostazioni di raffrescamento"},
gTw(){return"Ottimizza riscaldamento"},
gTx(){return"Sfrutta l'energia prodotta per riscaldare"},
gTs(){return"Configura riscaldamento"},
gTt(){return"Modifica le impostazioni di riscaldamento"},
ga8B(){return"sensore"},
ga8z(){return"Seleziona il sensore di energia del clima. Se non hai un sensore dedicato alla climatizzazione, seleziona il sensore dei consumi generale.\nPer una lettura pi\xf9 precisa dei consumi della climatizzazione \xe8 preferibile un sensore dedicato."},
ga8y(){return"Stai configurando le opzioni di *raffrescamento*"},
ga8A(){return"Stai configurando le opzioni di *riscaldamento*"},
ga2o(){return"selezione aree"},
ga2j(){return"Non hai configurato aree sul tuo Home Assistant. Crea almeno un'area da Home Assistant prima di procedere"},
ga2i(){return"Sei sicuro di voler interrompere l'inizializzazione? Le modifiche non salvate andranno perse."},
ga2n(){return"Definisci le Aree per ogni dispositivo clima. Inizializzando i dispositivi, Homsai raggruppa quelli nella stessa area per gestire al meglio la temperatura della casa."},
ga2k(){return"Area"},
ga2m(){return"Seleziona un'area"},
ga2l(){return"Cerca area..."},
ga2h(){return"Aggiungi area"},
gacC(){return"tipologia"},
gacx(){return"Seleziona il tipo di apparato da ottimizzare."},
gacz(){return"Radiante a pavimento"},
gacB(){return"Climatizzatore"},
gacA(){return"Termosifone"},
gacy(){return"Radiatore elettrico"},
gRC(){return"Consumo e produzione di oggi"},
gabe(){return"Questo grafico mostra l\u2019andamento della produzione e dei consumi di energia, di oggi. Si possono osservare i valori fino ad adesso registrati e quelli stimati fino a domani. In base alle prestazioni medie dell\u2019impianto fotovoltaico Homsai stima quanta energia verr\xe0 prodotta in funzione delle previsioni meteo raccolte. Questo permette di agire per tempo sulla climatizzazione e sui consumi, favorendo l\u2019autoconsumo piuttosto che l\u2019acquisto di energia dalla rete. Grazie alla previsione di produzione energetica potrai risparmiare sui costi dell\u2019energia."},
gabd(){return"Produzione"},
ga4N(){return"Consumo"}}
var z=a.updateTypes([]);(function inheritance(){var x=a.inherit
x(A.zN,C.m8)})()
B.b7w(b.typeUniverse,JSON.parse('{"zN":{"m8":[]}}'))
var y={c:"Compilando i seguenti campi Homsai sar\xe0 in grado di mostrarti quanto puoi ottimizzare i consumi e calcolarti il risparmio a fine mese.\n"}}
$__dart_deferred_initializers__["NuO0cWRbaN2Wozwv+gfGCYklMYY="] = $__dart_deferred_initializers__.current


homsai/rootfs/var/www/web/main.dart.js_1.part.js
self.$__dart_deferred_initializers__=self.$__dart_deferred_initializers__||Object.create(null)
$__dart_deferred_initializers__.current=function(a,b,c,$){var A={
byh(){B.D8("en")
return new A.zM()},
zM:function zM(){}},C,B
A=a.updateHolder(c[3],A)
C=c[5]
B=c[0]
A.zM.prototype={
gR2(){return"Ok"},
gnG(d){return"Confirm"},
gRX(){return"Retry"},
gjk(d){return"Next"},
ga3y(d){return"Back"},
git(d){return"Cancel"},
ga4T(){return"Copied text"},
S1(d){return d+"\xb0"},
gBk(d){return"Save"},
ga9k(){return"Invalid email"},
ga9m(){return"Invalid password"},
ga9n(){return"Invalid plant name"},
ga9l(){return"Mail not recognized. Register for the Closed Beta and wait for the confirmation email."},
gacQ(){return"Thank you for signing up. Your account is pending approval. In the next few days you will receive an email with the login credentials"},
gacR(){return"Check your e-mail box to find your login credentials"},
gaaW(d){return"Password"},
ga7a(){return"Forgot password?"},
gHI(d){return"Register"},
gaa0(){return"Login"},
gac1(){return"Reset password"},
gTU(){return"settings"},
ga3a(){return"Do you have already an account?"},
gabF(){return"Sign In"},
gaa1(){return"welcome\nback %1home%1"},
gabE(){return"make\nyour home\n%1truly%1\nsmart"},
gQY(){return"First time on Homsai?"},
gac2(){return"reset\nthe %1password%1"},
gSo(){return"Enter instance Url"},
gacN(){return"Enter URL"},
gSn(){return"The local and remote URLs are used to make your Home Assistant system reachable.\n%1Learn more%1"},
ga7N(){return"Connect to the WI-FI!"},
ga7M(){return"To identify the local instance of *Home Assistant*, the smartphone must be connected to the same Wi-Fi as the home automation hub."},
ga7S(){return"Searching local instance"},
ga7T(){return"No connection detected"},
ga7U(){return"Scanning..."},
ga7O(){return"Scan"},
ga7P(){return"Url non valido, assicurati che rispetti il formato http://XX.XXX.XXX.XXX:XXXX"},
ga7R(){return"L\u2019indirizzo fornito non corrisponde a nessuna istanza Home Assistant"},
ga7Q(){return"L\u2019istanza non risulta raggiungibile. Verifica da Home Assistant e riprova"},
ga2C(){return"Give a name to your home"},
gNv(){return"Edit home"},
ga2x(){return y.c},
ga2w(){return y.c},
ga2B(){return"Home name"},
ga2A(){return"GPS Coordinates"},
ga2z(){return"Why do we need plant's position?"},
ga2y(){return"Homsai uses the position to obtain meteorological data to predict the variation in consumption and production of photovoltaics."},
ga9h(){return"welcome!"},
ga9i(){return"log in and configure"},
ga9j(){return"comfort and savings"},
ga99(){return"Homsai is the application that allows you to *optimize* the management of your smart home."},
ga9a(){return"Register at Homsai with your %l1Home Assistant%l1 credentials and in a few moments you will have your smart home configured."},
ga9b(){return"Thanks to Homsai, your home will take care of you by *optimizing* consumption and allowing you to *save* on your bill."},
ga9d(){return"Thanks to our *artificial intelligence* you will receive personalized suggestions for *efficiency and savings*. With a simple click Homsai will take care of everything."},
ga9e(){return"We use %l2the most advanced technologies%l2 for data protection. Your privacy is secure."},
ga9f(){return"The intuitive interface will also allow you to control *all your devices* and set *routines* easily."},
ga96(){return"How does it work?"},
ga97(){return"What advantages do I have?"},
ga98(){return"I want to try Homsai"},
ga9g(){return"Skip"},
ga9c(){return"Step %1 of 3"},
ga7G(){return"https://www.home-assistant.io"},
ga8v(){return"https://homsai.app"},
ga37(){return"No internet connection"},
ga36(){return"The features are not available reconnect to continue using the app."},
ga35(){return"Home Assistant connection absent"},
ga34(){return"The features are not available reconnect from the settings to continue using the app."},
ga7X(){return"Consumption of yesterday"},
ga8h(){return"Optimized consumption"},
ga8i(){return"Purchased Energy"},
ga8g(){return"Energy Injected"},
ga7V(){return"--"},
ga7W(){return"Balance"},
ga8f(){return"Earn with Homsai"},
ga8e(){return"Consumption optimization"},
ga83(){return"What are you seeing"},
ga82(){return"The Optimized Consumption widget shows the difference in consumption within a day with and without optimization, discovering the economic advantage of using Homsai."},
ga85(){return"Current consumption"},
ga84(){return"Shows the trend of the real consumption of your home yesterday measured with the configured consumption sensors."},
ga87(){return"Optimized consumption"},
ga86(){return"On the basis of the consumption shown in 'Current consumption', Homsai shows the estimate of the values \u200b\u200bobtainable thanks to its optimization of consumption, taking into account the weather to efficiently manage air conditioning and predict energy production. \nThanks to its algorithms, Homsai is able to improve the efficiency of this optimization over time, exploiting more and more the energy produced by photovoltaic panels and using less purchased energy."},
ga89(){return"Legend"},
ga88(){return"Lines"},
ga7Y(){return": Energy produced by photovoltaic panels"},
ga7Z(){return": Total energy consumed"},
ga8a(){return"Areas"},
ga8_(){return": residual energy produced"},
ga80(){return": consumption of energy produced"},
ga81(){return": consumption of purchased energy"},
ga8b(){return"Balance = Energy Injected * - Energy Purchased *"},
ga8c(){return"Earnings with Homsai = Estimated Balance - Current Balance"},
ga8d(){return"* The cost of energy is calculated on the average price of energy considering \u20ac 0.35 for the one purchased and \u20ac 0.04 for the energy injected."},
gFy(){return"kW"},
gOP(){return"Solar panels"},
gOM(){return"Consumption"},
ga5N(){return"Available"},
gON(){return"Self-consumption"},
gOO(){return"Purchased"},
ga5O(){return"Loading data..."},
ga5M(){return"We are unable to retrieve the graph data, check in the settings that you have entered the correct data on the sensors."},
ga2K(){return"Configure system"},
gRD(){return"Production Sensor"},
gabf(){return"Choose production sensor"},
gOe(){return"Consumption Sensor"},
ga4O(){return"Choose consumption sensor"},
gRx(){return"Photovoltaic installation date"},
gHw(){return"MM/YYYY"},
gHx(){return"Photovoltaic nominal power"},
gRy(){return"kWp"},
gaaY(){return"Invalid date"},
gaaZ(){return"Complete the date correctly"},
gade(){return"Y"},
gvH(){return"No sensor selected"},
gaau(){return"M"},
ga3B(){return"Battery Sensor"},
ga3A(){return"Choose photovoltaic battery"},
ga8x(){return"HVAC Sensor"},
ga8w(){return"Choose HVAC consumption sensor"},
gVx(){return"Boiler Switch"},
gVw(){return"Choose a switch"},
gVv(){return"Select the boiler control to allow Homsai to optimize the heating."},
ga2J(){return"Incomplete configuration"},
ga2H(){return"Make sure you have filled in all the fields correctly. To view the optimized consumption correctly, complete the system configuration. You can still complete the configuration later from the Settings page."},
ga2I(){return"Incomplete fields:"},
gEN(){return"Ignore"},
gEO(){return"Modify"},
gNw(){return"Changes not saved. Do you want to ignore and go back or continue editing?"},
gaa3(){return"Log out"},
gaa2(){return"Do you really want to get out of Homsai?\nUpon exiting, all system data will be removed."},
gaa4(){return"For security reasons, it is not possible to log out without a connection to your Home Assistant"},
ga43(){return"Optimizer disabled. Updating the target temperature, you will able it again. Do you want to proceed?"},
ga9X(){return"Insert local Url"},
gabH(){return"Insert remote Url"},
gIG(){return"General"},
ga29(){return"Report a problem"},
ga28(){return"Help us improve Homsai"},
gRi(){return"Unable to reach %1"},
ga5Z(){return"Delete account"},
ga5Y(){return"Remove your account from Homsai"},
gTA(){return"Self-consumption optimization"},
gTz(){return"Make the most of the energy produced"},
gNn(){return"Email"},
ga2d(){return"Production sensor"},
ga2a(){return"Consumption sensor"},
ga27(){return"Battery sensor"},
ga2b(){return"Plant name"},
ga2c(){return"Plant position"},
ga2e(){return"App version"},
gO6(){return"Climate"},
ga44(){return"- 0,5"},
ga45(){return"+ 0,5"},
gaaJ(){return"Other settings"},
ga62(){return"Initialization..."},
ga46(){return"climate"},
ga4o(){return"Optimize device"},
ga47(){return"Allows you to optimize your device"},
ga4p(){return"Manual control"},
ga48(){return"Switch to manual until 11:59 pm"},
ga4q(){return"Limit time"},
ga49(){return"Optimize climate only in a certain time slot"},
ga4r(){return"From"},
ga4a(){return"8:00"},
ga4s(){return"To"},
ga4b(){return"8:00"},
ga4u(){return"Time not valid"},
ga4t(){return"To limit the operation of the optimizer it is necessary to set both the start time and the end time. Check that you have set both times correctly."},
ga4i(){return"The time limit cannot involve more days. The limitation end time must be after the start time."},
ga4j(){return"Valid example: \nFrom: 8:00 \nAt: 20:00"},
ga4k(){return"Not valid example: \nFrom: 20:00 \nAt: 8:00"},
ga4m(){return"Set the time"},
ga4l(){return"From hourly limit"},
ga4n(){return"To hourly limit"},
ga4h(){return"Inactive optimizer"},
ga4g(){return"The changes to the settings will not take effect until the optimizer is reactivated."},
ga4f(){return"Optimize"},
ga4c(){return"You are controlling "},
ga4d(){return"all climate devices."},
ga4e(){return"Invalid time"},
ga8W(){return"Install Add-On"},
ga8U(){return"The add-on for Home Assistant allows you to optimize consumption due to air conditioning, based on the energy production of the photovoltaic panels."},
ga8V(){return"Lorem ipsum ipsum Lorema"},
ga8R(){return"Automatic installation"},
ga8S(){return"Manual installation"},
ga8T(){return"Install later"},
gaa7(){return"Open Home Assistant from your computer or smartphone app."},
gaa8(){return"From the side menu, select the 'Settings' item"},
gaa9(){return"Select 'Add-ons', then 'Collection of add-ons' at the bottom."},
gaaa(){return"Open the menu at the top right, then select 'Repository'."},
gaab(){return"In the screen that opens, paste the following link and press 'Add'."},
gaac(){return"\u201cHomsai Engine\u201c will be added to the local component list."},
gaad(){return"Select \u201cHomsai Engine\u201c and press \u201cInstall\u201c then press \u201cStart\u201c to activate the add-on."},
ga3t(){return"Installation in progress..."},
ga3u(){return"The installation of the Homsai Engine can take up to 2 minutes. Keep the app open so you don't lose your progress."},
ga3v(){return"Installation completed"},
ga3s(){return"Installation falied"},
ga3p(){return"Check repository"},
ga3q(){return"Add-on installation"},
ga3o(){return"Boot add-on"},
ga3r(){return"Test"},
gGg(){return"Homsai Engine"},
ga33(){return"Incomplete sensor configuration."},
ga32(){return"Configure"},
ga2X(){return"No add-ons found. Install the add-on to be able to use Homsai."},
ga2W(){return"Install"},
ga3_(){return"Devices must be initialized in order to start the consumption optimizer."},
ga2Z(){return"Start"},
ga31(){return"Initialization"},
ga30(){return"Initializing devices..."},
ga2Y(){return"*Homsai Engine* is not reachable."},
ga2V(){return"Details"},
ga2U(){return"Start Homsai Engine"},
ga2T(){return"*Homsai Engine* has been disabled."},
ga2R(){return"Activate"},
ga2S(){return"Activating..."},
ga38(){return"*Homsai Engine Optimizer* has been disabled."},
ga8u(){return"homsai engine"},
ga8r(){return"Handle Add-on"},
ga8s(){return"Start wizard install"},
ga8t(){return"Add-on already installed"},
ga8q(){return"initialization"},
ga8p(){return"Initialization is necessary to identify which devices actively contribute to air conditioning.\nInitialization takes 20 minutes for each device, during which it will not be possible to control the device manually."},
ga8k(){return"Initialize devices"},
ga8m(){return"Starting\u2026"},
ga8l(){return"Initialization in progress"},
ga8j(){return"Estimated time"},
ga8o(){return"Remaining time"},
ga8n(){return"Initialized devices"},
gTy(){return"self-consumption"},
gTp(){return"CLIMATE"},
gTu(){return"Optimize cooling"},
gTv(){return"Use the energy produced to cool"},
gTq(){return"Configure cooling"},
gTr(){return"Change the cooling settings"},
gTw(){return"Optimize heating"},
gTx(){return"Use the energy produced to heat"},
gTs(){return"Configure heating"},
gTt(){return"Change your heating settings"},
ga8B(){return"sensor"},
ga8z(){return"Select the climate energy sensor. If you don't have a dedicated climate sensor, select the general consumption sensor.\nFor a more precise reading of air conditioning consumption, a dedicated sensor is preferable."},
ga8y(){return"You are configuring the *cooling* options"},
ga8A(){return"You are configuring the *heating* options"},
ga2o(){return"Select areas"},
ga2j(){return"You have not configured any areas on your Home Assistant. Create at least one area from Home Assistant before proceeding"},
ga2i(){return"Are you sure you want to abort initialization? Unsaved changes will be lost."},
ga2n(){return"Define the Areas for each climate device. By initializing the devices, Homsai groups those in the same area to better manage the temperature of the house."},
ga2k(){return"Area"},
ga2m(){return"Select area"},
ga2l(){return"Search area..."},
ga2h(){return"Add area"},
gacC(){return"typology"},
gacx(){return"Select the type of device to be optimized."},
gacz(){return"Radiant Floor Heater"},
gacB(){return"Split"},
gacA(){return"Radiator"},
gacy(){return"Eletric Radiator"},
gRC(){return"Consumption and production today"},
gabe(){return"This graph shows the trend of energy production and consumption today. You can observe the values recorded up to now and those estimated up to tomorrow. Based on the average performance of the photovoltaic system, Homsai estimates how much energy will be produced based on the weather forecasts collected. This allows you to act in time on air conditioning and consumption, favoring self-consumption rather than the purchase of energy from the grid. Thanks to the energy production forecast you can save on energy costs."},
gabd(){return"Production"},
ga4N(){return"Consumption"}}
var z=a.updateTypes([]);(function inheritance(){var x=a.inherit
x(A.zM,C.m8)})()
B.b7w(b.typeUniverse,JSON.parse('{"zM":{"m8":[]}}'))
var y={c:"By filling in the following Homsai will be able to tell you how much you can optimize the fields showing consumption and calculate your savings at the end of the month.\n"}}
$__dart_deferred_initializers__["/F67Fb3equPQZpdPE/QJbTQGyrk="] = $__dart_deferred_initializers__.current
